maps/map03.txt

```diff
@@ -1,7 +1,7 @@
-######
-#   .#
-# ## #
-# ## #
-# ## #
-# o  #
-######
+##########
+#o.......#
+#........#
+#........#
+#........#
+#.......g#
+##########
```

Comment:
Suddenly, this level is harder then it looks like. Maybe this is because of the ghost how constantly chasing you and you have to learn its behavior to avoid the ghost effectively.